config/config.go

```diff
@@ -165,6 +167,32 @@ func (cfg *AuctionTimeouts) validate(errs []error) []error {
 	return errs
 }
 
+type Compression struct {
+	Request  CompressionConfig `mapstructure:"request"`
+	Response CompressionConfig `mapstructure:"response"`
+}
+
+// CompressionConfig defines if compression is enabled and what type of compression to use
+type CompressionConfig struct {
+	Enabled bool   `mapstructure:"enabled"`
+	CType   string `mapstructure:"type"`
+}
+
+const CompressionTypeGZIP = "gzip"
```

Comment:
As mentioned [here](https://github.com/prebid/prebid-server/pull/2734/files#r1181359744), this will be changed to array.  How about adding new type `CompressionType`

```
type CompressionType string
const (
   gzip CompressionType = "gzip"
   brotli CompressionType = "brotli"
)

```

Reply:
Yes, I am adding that in my changes :)

Reply:
I just recommend to use the prefix as the enum type to group the values alphabetically. So, `compressionGzip` and `compressionBrotli`, etc...